hippynn/databases/xyz_database_tools.py

```diff
@@ -0,0 +1,184 @@
+from __future__ import annotations
+
+import os
+from pathlib import Path
+from typing import Optional, Tuple, Union
+
+import numpy as np
+import torch
+from ase import Atoms
+from ase.io import write as ase_write
+
+from hippynn.databases import NPZDatabase
+from hippynn.databases.h5_pyanitools import PyAniFileDB
+
+
+def load_base_database(
```

Comment:
This looks like a good new feature that maybe we should put in a more central location. It would be highly convenient if we could attempt to auto-detect other kinds as well, such as SNAP or ASE databases, so that code changes were minimized when moving between database formats.